numpydoc/tests/test_main.py

```diff
@@ -117,7 +117,7 @@ def test_validate_perfect_docstring():
     assert exit_status == 0
 
 
-@pytest.mark.parametrize("args", [[], ["--ignore", "ES01", "SA01", "EX01"]])
+@pytest.mark.parametrize("args", [[], ["--ignore", "ES01,SA01,EX01"]])
```

Comment:
We are missing a test that passes `--ignore` multiple times and also the space delimited option:

```suggestion
@pytest.mark.parametrize(
    "args",
    [[], ["--ignore", "ES01,SA01,EX01"], ["--ignore", "ES01", "--ignore", "SA01 EX01"]],
)
```